Framework/Kernel/inc/MantidKernel/Material.h

```diff
@@ -222,6 +227,7 @@ class MANTID_KERNEL_DLL Material final {
   double m_totalScatterXSection;
 
   boost::optional<AttenuationProfile> m_attenuationOverride;
+  boost::optional<AttenuationProfile> m_xRayAttenuationOverride;
```

Comment:
Please could you call this m_xRayAttenuationProfile - similar to my previous comments about not referring to this as an override